src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -17,6 +17,7 @@
         <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
         <Targets>Build;GetPackageAssets</Targets>
         <OutputItemType>PkgProjAsset</OutputItemType>
+        <UndefineProperties>OSGroup;TargetGroup</UndefineProperties>
```

Comment:
This was due to the fact that PackageTargetRuntime was being set by those properties, and throwing when building the csproj->pkgproj references.